code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -134,7 +134,7 @@
 	/// The maximum amount of vines a plant can have at one time
 	var/max_vines = 4
 	/// How far away a plant can attach a vine to something
-	var/vine_grab_distance = 5
+	var/vine_grab_distance = 2 //SKYRAT EDIT - Original 5
```

Comment:
Too small, make it 4

Reply:
I'm making it 3, so they cant retreat deep into vines and spam it again, and hopefully reduce instances of cornerbanging.